### ISSUE TITLE
[dataio] pyshovel library imports (Trac #1637)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1629]
Try the well-known simclasses and recclasses for importing when starting up.

Fix the bug where we have to toggle the frame to get a newly loaded class to show up.

Migrated from https://code.icecube.wisc.edu/ticket/1637
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:13:10", 
    "description": "Try the well-known simclasses and recclasses for importing when starting up.\n\nFix the bug where we have to toggle the frame to get a newly loaded class to show up.", 
    "reporter": "david.schultz", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1550067190995086", 
    "component": "combo core", 
    "summary": "[dataio] pyshovel library imports", 
    "priority": "normal", 
    "keywords": "", 
    "time": "2016-04-17T13:59:28", 
    "milestone": "", 
    "owner": "david.schultz", 
    "type": "enhancement"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20160726T18:00:18: 
* david.schultz changed status from "new" to "closed"
* david.schultz commented:

> In changeset 2628:
> 
> ```text
> #!CommitTicketReference repository="IceTray" revision="2628"
> reload the frame automatically. fixes #1629
> ```


* david.schultz changed resolution from "" to "fixed"

**Comment by icecube-trac** [comment]
Trac update at 20190111T21:52:55: nwhitehorn commented:

> In changeset 169197:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="169197"
> reload the frame automatically. fixes #1629
> ```

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:13:10: 
* olivas commented:

> Milestone July 2016 deleted


* olivas changed milestone from "July 2016" to ""
